mscore/events.cpp

```diff
@@ -517,7 +517,6 @@ void ScoreView::keyPressEvent(QKeyEvent* ev)
                   return;
                   }
             }
-
```

Comment:
add back this line please.
In fact, I just realized that you might wonder why we insist on not having new lines or blank lines being removed :) There is a good reason for it. Sometimes, we need to go back months in the git history to find a bug, and we use blame. Here is the blame view for this file https://github.com/musescore/MuseScore/blame/master/mscore/events.cpp
If you insert or delete lines, then, you will appear in the blame and it will be confusing for the person who will be trying to figure out a bug that has nothing to do with the commit message.